cloudify_cli/commands/uninstall.py

```diff
@@ -0,0 +1,85 @@
+########
+# Copyright (c) 2016 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+"""
+Handles 'cfy uninstall'
+"""
+
+from cloudify_cli.exceptions import CloudifyCliError
+from cloudify_cli import execution_events_fetcher
+from cloudify_cli import utils
+from cloudify_cli.commands import blueprints
+from cloudify_cli.commands import deployments
+from cloudify_cli.commands import executions
+
+
+def uninstall(blueprint_id, deployment_id, workflow_id, parameters,
+              allow_custom_parameters, timeout, include_logs):
+
+    # Although the `uninstall` command does not use the `force` argument,
+    # we are using the `executions start` handler as a part of it.
+    # As a result, we need to provide it with a `force` argument, which is
+    # defined below.
+    force = False
+
+    # if no workflow was supplied, execute the `uninstall` workflow
+    if workflow_id is None:
+        workflow_id = 'uninstall'
+
+    executions.start(workflow_id, deployment_id, timeout, force,
+                     allow_custom_parameters, include_logs, parameters)
+
+    # Because we want to delete the deployment right after executing the
+    # uninstall workflow, we may need to wait for the deployment executions to
+    # end.
+    wait_for_stop_dep_env_execution_to_end(deployment_id)
+
+    deployments.delete(deployment_id, ignore_live_nodes=False)
+
+    blueprints.delete(blueprint_id)
+
+# The function below (taken, with small modifications,
```

Comment:
Remove from here.
This is no longer relevant because of the `smart deployments` feature